entrypoint-invoicer.sh

```diff
@@ -0,0 +1,3 @@
+
+/bin/invoicer -lnd-host=$LNDHOST -lnd-invoice=/lnd/data/chain/bitcoin/mainnet/invoice.macaroon -lnd-readonly=/lnd/data/chain/bitcoin/mainnet/readonly.macaroon -mainnet -lnd-tls=/lnd/tls.cert
```

Comment:
`-mainnet` is unnecessary.

For merchants, we need a `users.list` file and it should be passed to a binary using `-users-file`.

Also bitcoind needs to have an `invoicer` user added and password should be passed here using `-bitcoind-pass`, other things might also need adjusting, see: https://github.com/lncm/invoicer/blob/master/bitcoind/client.go#L14-L17.

Is REST interface enabled on bitcoind? is `:8332` exposed there?

Also, it perhaps might make sense to split this one line into multiple.